### PR TITLE
Ansible 2.0 Support

### PR DESCRIPTION
**Pull request by careyrp**
Discovered three issues when moving to Ansible 2.0:
1. function parse_yaml_from_file has been replaced by method on DataLoader class
2. function path_dwim has been replaced by method on DataLoader class
3. list VALID_KEYS has been removed

For the functions, added try-catch block on the import. If 1.x ansible import fails, import ansible 2.0, and create parse_yaml_from_file and path_dwim functions that use the DataLoader class
For VALID_KEYS, I copied the definition from older ansible release to ansiblelint/utils.py.

### CODE REVIEW

**Comment by willthames** [comment]
I meant to reply to this earlier. I'm just a little worried about the magic VALID_TASKS - that it might become out of date. I had a quick look at v2 ansible and couldn't trivially determine how it's being done any more, but we should probably use that mechanism rather than keep using what will be a legacy set of task arguments.

**Comment by dmsimard** [comment]
Just figured I'd link the related issue this pull request is fixing: https://github.com/willthames/ansible-lint/issues/75

File: lib/ansiblelint/utils.py [hunk 1]
```diff
@@ -21,16 +21,36 @@
...
 import ansible.constants as C
 from ansible.module_utils.splitter import split_args
 import yaml
 from yaml.composer import Composer
 from yaml.constructor import Constructor
 
+try:
+    from ansible.utils import parse_yaml_from_file, path_dwim
+except ImportError:
+    from ansible.parsing import DataLoader
```
Comment (pabelanger) [inline lib/ansiblelint/utils.py]: this should be:

ansible.parsing.dataloader import DataLoader

**Comment by pabelanger** [comment]
Just tested this locally and with my comments it works with ansible 1.9.4 and 2.0. Any chance we can merge and get a release?

**Comment by willthames** [comment]
Merged with @pabelanger's fix.

Will upload the new module to pypi shortly

**Comment by willthames** [comment]
Thanks for all the hard work!

**Comment by willthames** [comment]
ansible-lint 2.3.0 is now available on pypi

File: lib/ansiblelint/utils.py (hunk 1 above)
Comment (plesiv) [inline lib/ansiblelint/utils.py]: `from ansible.parsing.dataloader import DataLoader` works from ansible commit [dcdcd9e9c518eafd533842664fe3c1545793a524](https://github.com/ansible/ansible/commit/dcdcd9e9c518eafd533842664fe3c1545793a524). If someone has odd version of ansible installed (from git).

**Pull request closed without merging**